scale.go

```diff
@@ -71,12 +71,12 @@ func (r *Scale[T]) Instance(next ...Next[T]) (*Instance[T], error) {
 }
 
 func (r *Scale[T]) Clear() {
-  for key := range r.activeModules {
-      r.activeModules[key].instantiatedModule.CloseWithExitCode(r.config.context, 0)
-      delete(r.activeModules, key)
-  }
+	for key := range r.activeModules {
+		r.activeModules[key].instantiatedModule.CloseWithExitCode(r.config.context, 0)
+		delete(r.activeModules, key)
+	}
```

Comment:
An entirely stylistic suggestion that doesn't functionally matter in the slightest.
```suggestion
	for key, module := range r.activeModules {
	        module.instantiatedModule.CloseWithExitCode(r.config.context, 0)
		delete(r.activeModules, key)
	}
```